test/chartmogul/customer.js

```diff
@@ -242,6 +242,89 @@ describe('Customer', () => {
         expect(res.has_more).to.eql(false);
       });
   });
+
+  it('creates a new opportunity from a customer', async () => {
+    const customerUuid = 'cus_00000000-0000-0000-0000-000000000000';
+    const postBody = {
+      customer_uuid: customerUuid,
+      owner: 'test1@example.org',
+      pipeline: 'New business 1',
+      pipeline_stage: 'Discovery',
+      estimated_close_date: '2023-12-22',
+      currency: 'USD',
+      amount_in_cents: 100,
+      type: 'recurring',
+      forecast_category: 'pipeline',
+      win_likelihood: 3,
+      custom: [{ key: 'from_campain', value: 'true' }]
```

Comment:
[NON-BLOCKING]

Does not affect the logic as it is all custom, but the word `campain` should be spelled `campaign` here and several other places in this file.